src/xb-builder-source.c

```diff
@@ -316,24 +317,26 @@ xb_builder_source_add_converter (XbBuilderSource *self,
 }
 
 /**
- * xb_builder_source_add_adapter:
+ * xb_builder_source_add_simple_adapter:
  * @self: a #XbBuilderSource
  * @content_types: mimetypes, e.g. `application/x-desktop,application/gzip`
  * @func: a callback, or %NULL
  * @user_data: user pointer to pass to @func, or %NULL
  * @user_data_free: a function which gets called to free @user_data, or %NULL
+ * @output_content_type: mimetype, e.g. `application/x-desktop`
  *
  * Adds a function that can be used to convert streams loaded with
  * xb_builder_source_load_xml().
  *
- * Since: 0.1.7
+ * Since: 0.1.15
  **/
 void
-xb_builder_source_add_adapter (XbBuilderSource *self,
-			       const gchar *content_types,
-			       XbBuilderSourceAdapterFunc func,
-			       gpointer user_data,
-			       GDestroyNotify user_data_free)
+xb_builder_source_add_simple_adapter (XbBuilderSource *self,
+				      const gchar *content_types,
+				      XbBuilderSourceAdapterFunc func,
+				      gpointer user_data,
+				      GDestroyNotify user_data_free,
+				      const gchar *output_content_type)
```

Comment:
I can't see a use case for the `output_content_type` -- if it's simple it's always going to be `application/xml`, no? You could just set a `gboolean` in `XbBuilderSourceAdapter`

Reply:
Done... Thanks!